peachjam/resources.py

```diff
@@ -150,12 +152,21 @@ def after_import_row(self, row, instance, row_number=None, **kwargs):
                 judge, _ = Judge.objects.get_or_create(name=judge)
                 judgment.judges.add(judge)
 
-        for case_number in row["case_numbers"]:
-            if case_number["number"]:
-                MatterType.objects.get_or_create(name=case_number["matter_type"])
-                CaseNumber.objects.create(
-                    document=judgment,
-                    number=case_number["number"],
-                    year=case_number["year"],
-                    matter_type=MatterType.objects.get(name=case_number["matter_type"]),
-                )
+        if "string_override" in row:
+            CaseNumber.objects.create(
+                string_override=row["string_override"], document=judgment
+            )
+        elif "case_numbers" in row:
+            for case_number in row["case_numbers"]:
+                if case_number["number"]:
+                    MatterType.objects.get_or_create(name=case_number["matter_type"])
+                    CaseNumber.objects.create(

```

Comment:
Just throw the string override (if present), into this object and do away with line 155-158. It's fine if it's null or empty. The code will only use it if it's valid.

Reply:
Basically, they're not exclusive. The SCA dataset has both the override and the breakdown, and we want to keep both for different purposes.